src/templates/import.js

```diff
@@ -694,14 +694,28 @@ module.exports = (router) => {
         template = JSON.parse(template);
       }
 
-      template = hook.alter('importOptions', template, req, res);
-      importTemplate(template, alters, (err, data) => {
-        if (err) {
-          return next(err.message || err);
-        }
-
-        return res.status(200).send('Ok');
+      const isTemplateValid = Object.values(template.forms).concat(Object.values(template.resources)).every(form=>{
```

Comment:
I would rather make this so that it is "smart" and allows for missing resources as long as there is a resource with this name that already exists in the database.